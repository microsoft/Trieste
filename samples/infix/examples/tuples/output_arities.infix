print "arity 0" (,);
print "arity 1" (1,);
print "arity 1 op" (2-1,);
print "arity 2" (1, 2);
print "arity 2 trailing" (1, 2,);
print "arity 3" (1, 2, 3);
print "arity 3 trailing" (1, 2, 3,);
print "arity 3 op" (1, 3-1, 1+4-2);