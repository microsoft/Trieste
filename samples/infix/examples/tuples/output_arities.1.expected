//! --enable-tuples --tuples-require-parens calculate
//! --enable-tuples --tuples-require-parens --use-parser-tuples calculate
//! --enable-tuples calculate
arity 0 <- (,)
arity 1 <- (1,)
arity 1 op <- (1,)
arity 2 <- (1, 2,)
arity 2 trailing <- (1, 2,)
arity 3 <- (1, 2, 3,)
arity 3 trailing <- (1, 2, 3,)
arity 3 op <- (1, 2, 3,)
